src/components/ScriptEditor.jsx

```diff
@@ -206,14 +222,30 @@ class ScriptEditor extends React.Component {
     const { name } = this.props;
     const text = this.getValue();
     const segmentInfo = getCharCount(text);
+    // const sampleContact = this.props.sampleContact;
+    // console.log("RENDER METHOD INSIDE SCRIPT EDITOR");
+    // console.log("PROPS");
+    // console.log(this.props);
+    // console.log("PROPS - script Text");
+    // console.log(text);
+    // console.log("PROPS - sampleContact");
+    // console.log(this.props.sampleContact);
+    // console.log("THIS.renderAdminTextPreview()");
+    // console.log(this.renderAdminTextPreview());
     return (
       <div>
         <div style={segmentInfo.charCount > 1600 ? { color: "red" } : {}}>
           Total characters: {segmentInfo.charCount}
+         
+         {/* {this.renderAdminTextPreview()} */}
+         
           {segmentInfo.charCount > 1600 ? (
             <span> Exceeded MMS maximum </span>
           ) : null}
         </div>
+          <div style={styles.editor}>
+  
+          </div>
```

Comment:
I should remove these.  I think this was from some initial attempts to update the ui.